### PR TITLE
Add topography grid for Southern Africa

### PR DESCRIPTION
**Pull request by santisoler**
Change template placeholders in `environment.yml` and `README.md`. Add required
dependencies in `environment.yml` and remove unused ones. Delete the `raw`
directory since the raw file is downloaded from NOAA using Pooch. Edit
`prepare.ipynb` notebook: read raw file, apply needed changes (described in the
`README.md`), export a new compressed file and generate a new `preview.jpg`
file.

### CODE REVIEW

**Comment by santisoler** [comment]
@leouieda I need the South Africa topography grid to be included in Ensaio so I can make Harmonica to use it and ditch the self-hosted data files. Let me know this looks go so I can generate a Zenodo entry for it. Once it's done, I can create a new function in Ensaio to fetch it and then make the changes in Harmonica.

**Comment by leouieda** [comment]
@santisoler as we discussed in our meeting, I would recommend making this a finer resolution (1 or 2 arc-minute if possible).

**Comment by santisoler** [comment]
@leouieda after what we discussed today in Transform22 Sprint, the grid is now generated by cropping the full ETOPO1 global grid to the Southern Africa region.

Let me know if you spot some needed changes. If not, feel free to merge. Once that's done, we need to make a release and upload the generated `.nc` file, right?

File: README.md [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
-
-![CHANGEME: The alt text of the image.](preview.jpg)
+![Southern Africa topography](preview.jpg)
 
 | | Summary |
 |--:|:--|
-| File | `CHANGEME.csv` |
-| Size | CHANGEME Mb |
-| Version | [CHANGEME](https://github.com/fatiando-data/CHANGEME/releases/latest) |
+| File | `southern-africa-topography.csv` |
```
Comment (leouieda) [inline README.md]:
```suggestion
| File | `southern-africa-topography.nc` |
```

**Comment by leouieda** [comment]
Just a small fix: file extension was `.nc` instead of `.csv`. Merging in!

**Comment by santisoler** [comment]
Nice catch! Thanks!

**Pull request merged**